build.gradle

```diff
@@ -35,6 +35,8 @@ version = "0.0.0-SNAPSHOT"
 // url to the snapshot repository of a private maven repo.
 def toolkitSnapshotRepo = "http://104.197.230.53:8081/nexus/content/repositories/snapshots/"
 
+def protoVersion = "3.0.0-beta-3"
```

Comment:
There are a bunch of double-quoted strings in the file; I think it makes sense to do a separate pass for that.